src/classes/RD2_OpportunityService_TEST.cls

```diff
@@ -199,6 +199,33 @@ private with sharing class RD2_OpportunityService_TEST {
         testAndAssertOppService(rd, expectedOppName);
     }
 
+    /***
+    * @description Verifies the installment opportunities are created with the record types set in the NPSP setting
+    */
+    @isTest
+    private static void oppRecordTypeShouldMatchWithSetting() {
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+
+        Id defaultRecordTypeId = UTIL_RecordTypes.getRecordTypeIdForGiftsTests(Opportunity.sObjectType);
+        UTIL_CustomSettingsFacade.getRecurringDonationsSettingsForTest(new npe03__Recurring_Donations_Settings__c(
```

Comment:
- Can we utilize existing `createRdSettingsWithStage()` to set the default record type Id on installment Opp? For example, pass a new argument `oppRecordTypeId`? Change is similar to `createRecurringDonation()` helper method change.
- Set `npe03__Open_Opportunity_Behavior__c` in the `createRdSettingsWithStage()` method.